lackey/InputEmulation.py

```diff
@@ -66,6 +66,7 @@ def moveSpeed(self, location, seconds=0.3):
         original_location = mouse.get_position()
         mouse.move(location.x, location.y, duration=seconds)
         if mouse.get_position() == original_location and original_location != location.getTuple():
+            self._lock.release()
```

Comment:
This can be simplified by moving it above the if statement and eliminating line 74.

Reply:
Hi.
If line 74 "self._lock.release()" is removed, the lackey stops working.
The solution to insert self._lock.release() before the IOError raise was great.
Thank you again!

Reply:
Hi.
I'm sorry, but
I hadn't tried moving line 69 out of the if.
I tested it again. Deleting line 74 worked.